oak/server/rust/oak_runtime/src/runtime/introspect.rs

```diff
@@ -150,10 +151,11 @@ async fn make_server(
     info!("starting introspection server on {:?}", addr);
 
     // And a MakeService to handle each connection...
+    let runtime_clone = runtime.clone();
     let make_service = make_service_fn(move |_| {
-        // The `Arc<Runtime>` is moved into this closure, but needs to be cloned
+        // The runtime_clone is moved into this closure, but needs to be cloned
         // because this closure is called for every connection.
-        let runtime = runtime.clone();
+        let runtime = runtime_clone.clone();
```

Comment:
I'll admit this isn't the most beautiful naming, but essentially I need to clone the runtime initially, so that `runtime` isn't moved into the closure and can be used later for the introspection events.